### PR TITLE
Begin the steps to release 1.0.15

### PR DESCRIPTION
**Pull request by abersnaze**
Can get two more :+1: from the committers.

### CODE REVIEW

**Comment by akarnokd** [comment]
It would be great to get these PRs in as well:

#3458 : CompositeSubscription javadoc change
#3455 : [Bug] OnErrorFailedException propagation fix
#3454 : [Bug] Fix bounded replay() request behavior
#3438 : Better null tolerance, although this may need rebase due to #3455

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,44 @@
...
 
+### Version 1.0.15 – October 9 2015 ([Maven Central](http://search.maven.org/#artifactdetails%7Cio.reactivex%7Crxjava%7C1.0.15%7C)) ###
+
+* [Pull 3448] (https://github.com/ReactiveX/RxJava/pull/3448) Single delay
+* [Pull 3429] (https://github.com/ReactiveX/RxJava/pull/3429) Removed the alias BlockingObservable#run
+* [Pull 3417] (https://github.com/ReactiveX/RxJava/pull/3417) Add Single.doOnSuccess()
+* [Pull 3418] (https://github.com/ReactiveX/RxJava/pull/3418) Add Single.fromCallable()
+* [Pull 3419] (https://github.com/ReactiveX/RxJava/pull/3419) Add Single.doOnError()
+* [Pull 3423] (https://github.com/ReactiveX/RxJava/pull/3423) Renaming Observable#x to Observable#extend
+* [Pull 3422] (https://github.com/ReactiveX/RxJava/pull/3422) Update README.md
```
Comment (benjchristensen) [inline CHANGES.md]: This line isn't needed since it's not actually affecting code or functionality.

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,44 @@
...
+* [Pull 3174] (https://github.com/ReactiveX/RxJava/pull/3174) Blocking subscribe methods for convenience
+* [Pull 3351] (https://github.com/ReactiveX/RxJava/pull/3351) Make BlockingOperatorToIterator exert backpressure.
+* [Pull 3357] (https://github.com/ReactiveX/RxJava/pull/3357) Eager ConcatMap
+* [Pull 3342] (https://github.com/ReactiveX/RxJava/pull/3342) Remove redundant onStart implementation in OperatorGroupBy
+* [Pull 3361] (https://github.com/ReactiveX/RxJava/pull/3361) Safer error handling in BlockingOperatorToFuture
+* [Pull 3363] (https://github.com/ReactiveX/RxJava/pull/3363) Remove unused private method from CachedObservable and make "state" final
+* [Pull 3408] (https://github.com/ReactiveX/RxJava/pull/3408) DoOnEach: report both original exception and callback exception.
+* [Pull 3386] (https://github.com/ReactiveX/RxJava/pull/3386) 1.1.0 - Changed Observable.doOnRequest(Action1) to `@Experimental` from `@Beta`
```
Comment (benjchristensen) [inline CHANGES.md]: Isn't this title wrong? We left it as beta, just updated docs.

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,44 @@
...
+* [Pull 3415] (https://github.com/ReactiveX/RxJava/pull/3415) pull back the Experimental/Beta of the changes until 1.1.x (+1 squashed commit)
```
Comment (benjchristensen) [inline CHANGES.md]: Generally I didn't include things like this in release notes that were maintenance or reversions of things that never had been released, as it just clutters the release notes and doesn't actually affect anyone.

File: CHANGES.md [hunk 4]
```diff
@@ -1,5 +1,44 @@
...
+* [Pull 3149] (https://github.com/ReactiveX/RxJava/pull/3149) Scheduler shutdown capability
+* [Pull 3384] (https://github.com/ReactiveX/RxJava/pull/3384) Fix for take() reentrancy bug.
+* [Pull 3356] (https://github.com/ReactiveX/RxJava/pull/3356) Fix to a bunch of bugs and issues with AsyncOnSubscribe
+* [Pull 3362] (https://github.com/ReactiveX/RxJava/pull/3362) Fix synchronization on non-final field in BufferUntilSubscriber
+* [Pull 3365] (https://github.com/ReactiveX/RxJava/pull/3365) Make field final and remove unnecessary unboxing in OnSubscribeRedo.RetryWithPredicate
+* [Pull 3383] (https://github.com/ReactiveX/RxJava/pull/3383) Fix typo in a comment inside Observable.subscribe
```
Comment (benjchristensen) [inline CHANGES.md]: I generally chose to not include comment/doc PRs in release notes.

**Comment by benjchristensen** [comment]
:+1: for getting this release out. I provided some non-essential feedback on suggestions for things I would change or not include in the release notes based on how I've done it in thus far. With or without those changes (except the one I thing has a misleading title), I'm good with the release going out.

**Comment by benjchristensen** [comment]
Agreed that we should get the items @akarnokd mentioned into this release. Some are already merged and just need to be put in the release notes. I've looked at the others and think they are all okay for merging before release.

**Comment by akarnokd** [comment]
Great, these are already merged in:

#3458 : CompositeSubscription javadoc change
#3455 : [Bug] OnErrorFailedException propagation fix
#3438 : Better null tolerance, although this may need rebase due to #3455

I feel this one can be merged but would like someone to review it in more detail:

#3454 : [Bug] Fix bounded replay() request behavior

I don't know how urgent are these as they were reported by Netflix:

#3428 GroupBy request fix
#3169 Merge support for horizontally unbounded mode

File: CHANGES.md (hunk 2 above)
Comment (stealthcode) [inline CHANGES.md]: Updating.

File: CHANGES.md (hunk 1 above)
Comment (stealthcode) [inline CHANGES.md]: Removing.

File: CHANGES.md (hunk 3 above)
Comment (stealthcode) [inline CHANGES.md]: Thanks. Removing.

File: CHANGES.md (hunk 4 above)
Comment (stealthcode) [inline CHANGES.md]: Ok.

**Comment by stealthcode** [comment]
:+1:

**Comment by stealthcode** [comment]
I think I'd prefer to cut a release of 1.0.15 asap and get the changes like #3454, #3428, and #3169 in a subsequent 1.0.16 release. 

Can I get a +1 to initiate the 1.0.15 release?

**Comment by artem-zinnatullin** [comment]
:+1: (not a "collaborator" though). Let's finally release 1.0.15, a lot of good stuff for Rx users!

@abersnaze can you please squash commits in this PR?

**Comment by benjchristensen** [comment]
+1 for release of what is already merged.

**Comment by akarnokd** [comment]
:+1:

**Comment by abersnaze** [comment]
I was getting lazy. thanks @artem-zinnatullin.

**Comment by stevegury** [comment]
:+1:

**Pull request merged**